node/deepgram_topic_detection/src/index.js

```diff
@@ -0,0 +1,33 @@
+const { Deepgram } = require("@deepgram/sdk");
+
+module.exports = async (req, res) => {
+  const apiKey = req.variables["DEEPGRAM_API_KEY"];
+  const fileUrl = req.payload["fileUrl"];
```

Comment:
It would be nice if payload were a string and it was parsed here rather than have the payload be a JSON object. This would help make sure the function is compatible with Appwrite as well.

Reply:
Yes, this will be better approach
I was also struggling while testing the current version in Appwrite function test
I will fix it.